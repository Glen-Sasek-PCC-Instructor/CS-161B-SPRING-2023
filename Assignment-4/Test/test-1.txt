6
4
12
3
-1
6
4
-1
