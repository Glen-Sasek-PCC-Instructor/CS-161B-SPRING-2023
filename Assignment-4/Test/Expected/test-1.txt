Welcome to my Set of Numbers program!!
Enter numbers for set 1 on separate lines (-1 to end):
>> >> >> >> >> Your set 1 with 4 numbers ordered:
3 4 6 12
Enter numbers for set 2 on separate lines (-1 to end):
>> >> >> Your set 2 with 2 numbers ordered:
4 6
The new sets are:
Set1: 3 12
Set2: 4 6
Thank you for checking out my Set of Numbers program!

